boards/arm/google_dragonclaw/google_dragonclaw.dts

```diff
@@ -91,13 +91,32 @@
 	};
 };
 
+&gpiob {
+	unused {
+		gpio-hog;
+		gpios = <2 0>, <5 0>;
```

Comment:
what are you trying to achieve with these? normally you'd specify either `GPIO_ACTIVE_HIGH` or `GPIO_ACTIVE_LOW` and a property like `output-high` or `output-low`.

Reply:
To save as much power as possible on STM32F412, the unused pins should be configured as analog. This is not a default state of the pins. Flag 0 means GPIO_DISCONNECTED (haven't found a proper dt define), the GPIO driver configures such pin as [analog](https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/gpio/gpio_stm32.c#L84). 
![image](https://github.com/zephyrproject-rtos/zephyr/assets/72437029/35737560-7d24-44ea-ab65-8c761f4e063b)
this structure explains why analog is the most power efficient mode - it disables Schmitt trigger. Please take a look at [1.2.6](https://www.st.com/resource/en/application_note/an4365-using-stm32f4-mcu-power-modes-with-best-dynamic-efficiency-stmicroelectronics.pdf) for more details.

Reply:
Can we use `#define GPIO_ACTIVE_HIGH        (0 << 0)` instead of 0? I think the GPIO will be disconnected if we don't provide any of `output-high`, `output-low`, `input`.

Reply:
Makes sense. I'll add a comment and use `GPIO_ACTIVE_HIGH`.